war/src/main/less/modules/icons.less

```
@@ -121,3 +121,9 @@
     animation: spin 1.5s linear infinite;
   }
 }
+
+.schedule-build-anime {
+  svg {
+    animation: fadeout 1.5s ease infinite;
+  }
+}
```

```suggestion
@keyframes pulse-animation {
  0%,
  100% {
    opacity: 1;
  }
  50% {
    opacity: 0;
  }
}

.pulse-animation {
  svg {
    animation: pulse-animation 2s ease infinite;
  }
}
```

Untested (like your version 👀 ) but it turns your proposal into an agnostic class that can be reused in other occasions, if needed, and makes the animation more calm. This change simulates a pulse, instead of having the icon pop up instant after the fadeout.

Needs to be adjusted in the jelly file too, if we want to use it.

---

This is way better. Thank you @NotMyFault !